tools/who_what_benchmark/tests/test_cli_image.py

```diff
@@ -38,13 +38,14 @@ def run_wwb(args):
 
 def setup_module():
     for model_id in OV_IMAGE_MODELS:
-        MODEL_PATH = os.path.join(MODEL_CACHE, model_id.replace("/", "--"))
+        MODEL_PATH = MODEL_CACHE / model_id.replace("/", "--")
```

Comment:
It seems that WWB and GenAI tests use different replacement strategies. `.replace("/", "_")` in GenAI